Meta/check-symbols.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+
+script_path=$(cd -P -- "$(dirname -- "$0")" && pwd -P)
+cd "$script_path/.."
+
+# The __cxa_guard_* calls are generated for (non trivial) initialzation of local static objects.
+# These symbols are OK to use within serenity code, but they are problematic in LibC because their
+# existence breaks ports (the implementation of these symbols resides in libstdc++.a which we do not link against in ports).
+# To eliminate the need for these symbols, avoid doing non-trivial construction of local statics in LibC.
+
+FORBIDDEN_SYMBOLS="__cxa_guard_acquire __cxa_guard_release
```

Comment:
You probably meant to end this string at EOL?

Reply:
huh :sweat_smile: